include/GafferScene/ObjectSource.h

```diff
@@ -100,14 +100,13 @@ class ObjectSource : public SceneNode
 		virtual IECore::ConstInternedStringVectorDataPtr computeSetNames( const Gaffer::Context *context, const ScenePlug *parent ) const;
 		virtual GafferScene::ConstPathMatcherDataPtr computeSet( const IECore::InternedString &setName, const Gaffer::Context *context, const ScenePlug *parent ) const;
 
+		virtual void hashStandardSetNames( const Gaffer::Context *context, IECore::MurmurHash &h ) const;
```

Comment:
It's a tiny thing, but let's keep the order the same as before, with sets below source. I generally try to keep everything ordered as it is within ScenePlug, so there's a natural pattern to each of the interfaces a class has to implement.